tags.go

```diff
@@ -54,3 +54,14 @@ func GetTagValue(tags []string, key string, separator string) string {
 
 	return ""
 }
+
+// TagExists returns true if tag exists in tags
+func TagExists(tags Tags, tagValue string) bool {
```

Comment:
I would call this function `HasTag` instead 🙂

Reply:
Can this not be a method also? You only need to pass the tagValue in then.